docs/marionette.view.md

```diff
@@ -882,6 +886,149 @@ view.addRegion("foo", "#bar");
 view.removeRegion("foo");
 ```
 
+### Binding To User Input
+
+Views can bind custom events whenever users perform some interaction with the
+DOM. Using the view `events` and `triggers` handlers lets us either bind  user
+input directly to an action or fire a generic trigger that may or may not be
+handled.
+
+#### View `events`
+
+The view `events` attribute binds DOM events to functions or methods on the
+view. The simplest form is to reference a method on the view:
+
+```javascript
+var Mn = require('backbone.marionette');
+
+var MyView = Mn.View.extend({
+  events: {
+    'click a': 'showModal'
+  },
+
+  showModal: function(event) {
+    console.log('Show the modal');
+  }
+});
+```
+
+The DOM event gets passed in as the first argument, allowing you to see any
+information passed as part of the event.
+
+**When passing a method reference, the method must exist on the View.**
+
+##### Defining Listeners
+
+Listeners are defined as:
+
+```javascript
+eventname jqueryselector
+```
+
+* The `eventname` part refers to a jQuery DOM event e.g. `click` or `change`.
+* The `jqueryselector` part is a jQuery selector string e.g. `.myclass`
+
+You can also pass just the eventname part causing the event handler to fire on
+the entire view. This is especially useful for buttons and click handlers:
+
+```javascript
+var Mn = require('backbone.marionette');
+
+var ButtonView = Mn.View.extend({
+  tagName: 'button',
+
+  events: {
+    click: 'showAlert'
+  },
+
+  showAlert: function() {
+    alert('Button was clicked');
+  }
+});
+```
+
+##### Passing a Function
+
+The `events` attribute can also directly bind functions:
+
+```javascript
+var Mn = require('backbone.marionette');
+
+var MyView = Mn.View.extend({
+  events: {
+    'click a': function(event) {
+      console.log('Show the modal');
+    }
+  }
+});
+```
+
+As when passing a string reference to a view method, the `events` attribute
+passes in the `event` as the argument to the function called.
+
+#### View `triggers`
+
+The view `triggers` attribute binds DOM events to Marionette View events that
+can be responded to at the view or parent level. For more information on events,
+see the [events documentation](./marionette.events.md). This section will just
+cover how to bind these events to views.
+
+```javascript
+var Mn = require('backbone.marionette');
+
+var MyView = Mn.View.extend({
+  triggers: {
+    'click a': 'link:clicked'
+  },
+
+  onLinkClicked: function() {
+    console.log('Show the modal');
+  }
+});
+```
+
+When the `a` tag is clicked here, the `link:click` event is fired. This event
+can be listened to using the
+[Magic Method Binding](./marionette.events.md#magic-method-binding) technique
+discussed in the [events documentation](./marionette.events.md).
+
+##### Defining Listeners
+
+Listeners are defined as:
+
+```javascript
+eventname jqueryselector
+```
```

Comment:
from [here](https://github.com/marionettejs/backbone.marionette/pull/3061/files#diff-754a14ba135eb5c5b22e5325d9d4092fR1001)
to [this line](https://github.com/marionettejs/backbone.marionette/pull/3061/files#diff-754a14ba135eb5c5b22e5325d9d4092fR1029) looks the same if no similar to 
https://github.com/marionettejs/backbone.marionette/pull/3061/files#diff-754a14ba135eb5c5b22e5325d9d4092fR926